src/uct/ib/rdmacm/rdmacm_md.c

```diff
@@ -106,6 +106,31 @@ static int uct_rdmacm_is_addr_route_resolved(struct rdma_cm_id *cm_id,
     return 1;
 }
 
+static int uct_rdmacm_is_sockaddr_inaddr_any(struct sockaddr *addr)
+{
+    struct sockaddr_in6 *addr_in6;
+    struct sockaddr_in *addr_in;
+
+    switch (addr->sa_family) {
+    case AF_INET:
+        addr_in = (struct sockaddr_in *)addr;
+        if (addr_in->sin_addr.s_addr == INADDR_ANY) {
+            return 1;
+        }
+        break;
+    case AF_INET6:
+        addr_in6 = (struct sockaddr_in6 *)addr;
+        if (!memcmp(&addr_in6->sin6_addr, &in6addr_any, sizeof(addr_in6->sin6_addr))) {
+            return 1;
+        }
+        break;
+    default:
+        ucs_error("Invalid address family");
```

Comment:
no need for error, just return 0. maybe just ucs_debug() and also print the address family number